README.md

```diff
@@ -68,6 +69,7 @@ BOOTSTRAP_AMOUNT=15.0
 HTTP_PROVIDER_HOST=127.0.0.1
 HTTP_PROVIDER_PORT=8545
 LOCKSMITH_PURCHASER_ADDRESS=0xe29ec42f0b620b1c9a716f79a02e9dc5a5f5f98a
+STRIPE_KEY=pk_test_BHXKmScocCfrQ1oW8HTmnVrB
```

Comment:
Its probably a moot point now, but we should not have out test stripe key here as a default

Reply:
The paywall crashes without it, and consumes all available memory. I'm open to other suggestions.

Reply:
We could put gibberish in there, just not what we are currently using

Reply:
You should be able to use something like `pk__test_not_a_real_key`, which will allow stripe to instantiate, but any requests made with it will fail.

> consumes all available memory
😳